core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -98,5 +158,13 @@
     {{ include_js_file('domain/editor/ChangeObjectFactory.js') }}
     {{ include_js_file('domain/editor/UndoRedoService.js') }}
     {{ include_js_file('domain/summaries/ExplorationSummaryBackendApiService.js') }}
+    {{ include_js_file('domain/utilities/UrlInterpolationService.js') }}
+
+    {{ include_js_file('exploration_editor/EditorServices.js') }}
+    {{ include_js_file('exploration_editor/ExplorationEditor.js') }}
+    {{ include_js_file('exploration_editor/GadgetValidationService.js') }}
```

Comment:
woah. Why do you need the exploration editor, the gadget validation service, the expression interpolation service, etc.? Collections don't have gadgets...